### PR TITLE
feat(MailchimpOperation): Add class and tests and interface

### LINKED ISSUE #1: feat(MailchimpOperation): Add class and tests and interface
Probably best to place in `src/core`

And use the namespace `CrazyFactory\MailChimp\Core`

### PR DESCRIPTION
**Pull request by lalita13008**
close #1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "type": "library",
     "autoload": {
         "psr-4": {
-            "CrazyFactory\\Shop\\Mailing\\": "src/"
+            "CrazyFactory\\Shop\\Mailing\\": "src/",
+            "CrazyFactory\\Shop\\Mailing\\Core\\": "src/core"
```
Comment (dacgray) [inline composer.json]: Doesn't this find classes in src/core anyways?

Comment (lalita13008) [reply composer.json]: Yes, it cannot found the class on travis

**Comment by dacgray** [comment]
Please remove the methods from the MailchimpOperation and the mock too ^^

File: composer.json (hunk 1 above)
Comment (dacgray) [reply composer.json]: `src/core/` ^^

**Pull request merged**